src/klein/_plating.py

```diff
@@ -96,54 +114,58 @@ def __init__(self, defaults=None, tags=None,
         """
         self._defaults = {} if defaults is None else defaults
         self._loader = TagLoader(tags)
-        self._presentation_slots = {self.CONTENT} | set(presentation_slots)
+        self._presentationSlots = {self.CONTENT} | set(presentation_slots)
 
-    def routed(self, routing, content_template):
+    def routed(self, routing, tags):
         """
         
         """
-        @wraps(routing)
         def mydecorator(method):
-            loader = TagLoader(content_template)
-            @routing
-            @wraps(method)
-            def mymethod(request, *args, **kw):
-                data = method(request, *args, **kw)
+            loader = TagLoader(tags)
+            @modified("plating route renderer", method, routing)
+            @bindable
+            @inlineCallbacks
+            def mymethod(instance, request, *args, **kw):
+                data = yield _call(instance, method, request, *args, **kw)
                 if _should_return_json(request):
                     json_data = self._defaults.copy()
```

Comment:
I'm seeing these 4 lines:

```
json_data = some_dict.copy()
for ignored in self._presentationSlots:
     json_data.pop(ignored_None)
return json_data
```

Pop up like 3 times. Sounds like this should be a method (`with_presentation_removed` maybe?)